src/validation.h

```diff
@@ -208,19 +208,26 @@ struct PackageMempoolAcceptResult
 };
 
 /**
- * Try to add a transaction to the mempool. This is an internal function and is
- * exposed only for testing. Client code should use ChainstateManager::ProcessTransaction()
+ * Try to add a transaction to the mempool. This is an internal function and is exposed only for testing.
+ * Client code should use ChainstateManager::ProcessTransaction()
  *
- * @param[in]  active_chainstate  Reference to the active chainstate.
+ * @param[in]  chainparams        Chain paramenters.
```

Comment:
This argument could be removed. `AcceptToMemoryPool()` could just take the `m_params` from the `active_chainstate` that's passed in.

Reply:
Done.
As this PR is removing the `CChainParams& chainparams` from ATMP (commit 863703c),  I also added a new commit to remove `CTxMemPool& pool` as suggested in https://github.com/bitcoin/bitcoin/pull/23437#issuecomment-962536149.